config/ftbquests/quests/chapters/mekanism.snbt

```diff
@@ -2520,16 +2520,18 @@
 			}]
 		}
 		{
+			title: "5th Tier Ore Processing"
 			icon: "mekanism:fluorite_gem"
 			x: -2.5d
 			y: 6.5d
 			subtitle: "Research Complete"
 			description: [
 				"The ultimate in ore processing efficiency… or is it? "
 				""
-				"Mekanism’s highest form of ore processing ends at quintupling and requires a hefty amount of infrastructure to access. As with quadrupling, the exact process can be determined from JEI by looking at metal crystals. "
+				"Mekanism’s highest form of ore processing ends here, at 3.33 ingot per raw ore (or 5 per ore block), and requires a hefty amount of infrastructure to access. As with last tier, the exact process can be determined from JEI by looking at metal crystals. Invest in some sort of auto-breaker of ores with Fortune, as this could boost per ore block yield way above 5 ingots!"
```

Comment:
As above, don't even mention the worse rates. We're talking about max output, not the full spectrum.